ios/FluentUI/Bottom Sheet/BottomSheetController.swift

```diff
@@ -134,6 +134,9 @@ public class BottomSheetController: UIViewController {
         }
     }
 
+    /// Indicates if the content should be hidden when the sheet is collapsed
```

Comment:
Setting this property should probably call updateExpandedContentAlpha().

On the demo app this happens when we change the toggle switch:



https://user-images.githubusercontent.com/68076145/140813905-4cb28420-d91f-47ca-8eb6-309838ae6f6d.mov


It should update right away when the property is set.